packages/boilerplate-generator-tests/browser_test.js

```diff
@@ -0,0 +1,100 @@
+import { parse, serialize } from 'parse5';
```

Comment:
I think the name of this file should be consistent with that of the appropriate `arch` template (i.e. `boilerplate_web.browser.template.js`), and it should be plural like our other test files (e.g. `web.browser-tests.js`, `web.cordova-tests.js`)